conf/nutch-default.xml

```diff
@@ -1874,6 +1874,72 @@ visit https://wiki.apache.org/nutch/SimilarityScoringFilter-->
   </description>
 </property>
 
+
+<!-- lib-htmlunit plugin properties; applies to protocol-htmlunit -->
+
+<property>
+  <name>htmlunit.page.load.delay</name>
```

Comment:
These properties are near duplicate. I would suggest that we consider sharing the properties as duplicating them seems a bit too much.